pkg/gatherers/workloads/gather_workloads_info.go

```
@@ -104,18 +129,24 @@ func gatherWorkloadInfo(
 ) ([]record.Record, []error) {
 	errs := []error{}
 
-	workloadInfos, runtimeInfoErrs := gatherWorkloadRuntimeInfos(ctx, coreClient, configClient)
-	errs = append(errs, runtimeInfoErrs...)
 
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)